CONTRIBUTING.md

```diff
@@ -93,6 +93,56 @@ The Continuous Integration (CI) runs automatically when a pull request is opened
 - Compilation: the code will be compiled using [Github Actions](https://github.com/features/actions) under Linux, Windows and MacOS, and the unit tests will be run;
 - SonarCloud will report the code smells, duplications and the code coverage. You have to fix all the relevant code smells and add unit tests to reach the required barrier.
 
+## Committer rights on one or several repositories
+
+### Committer rights
+
+A Committer is a developer who has the right to merge on protected branches on a given repository.
+A Committer may have rights on one or several repositories according to their expertise.
+
+A Committer is able to vote when there is someone asking for Committer's rights on a repository.
+
+A Committer is able to vote when there is a candidate for a TSC member seat.
+
+### Committer responsibilities
+
+**Being a Committer is a huge responsibility.**
```

Comment:
huge sounds overstated - great responsibility? Or turn it like the common saying, "with committer rights comes great responsibility." :spider: :superhero_man:

Reply:
"huge" changed to "significant"